pkg/gatewayserver/config.go

```diff
@@ -31,6 +31,7 @@ type BasicStationConfig struct {
 	FallbackFrequencyPlanID string `name:"fallback-frequency-plan-id" description:"Fallback frequency plan ID for non-registered gateways"`
 	Listen                  string `name:"listen" description:"Address for the Basic Station frontend to listen on"`
 	ListenTLS               string `name:"listen-tls" description:"Address for the Basic Station frontend to listen on (with TLS)"`
+	UseWSSTrafficEndpoint   bool   `name:"use-wss-traffic-endpoint" description:"Use WSS for the traffic endpoint regardless of the TLS setting"`
```

Comment:
Inconsistent TLS vs WSS in this struct and "endpoint" vs "address" like in other config; suggesting `UseTrafficTLSAddress`

Reply:
good one. Done